Hometask01.part02

maximlip opened the pull request:
Hometask 01. Part 02. Bash script.

maximlip committed:
Lesson 2 Part 1: Add game() function

Signed-off-by: Maxim Lipchanskyi <m4554747@gmail.com>

maximlip committed:
Lesson 2 Part 1: Add main() function

Added dummy main function

Signed-off-by: Maxim Lipchanskyi <m4554747@gmail.com>

maximlip committed:
hometask01.part02: Add file part2

Added file part2 for bash script

Signed-off-by: Maxim Lipchanskyi <m4554747@gmail.com>

maximlip committed:
hometask01.part02: Add mkdir functinality

Added creation folder tmp/guesanumber

Signed-off-by: Maxim Lipchanskyi <m4554747@gmail.com>

maximlip committed:
hometask01.part02: Add file copy

Added file dz1.c copy to folder tmp/guesanumber

Signed-off-by: Maxim Lipchanskyi <m4554747@gmail.com>

maximlip committed:
hometask01.part02: Add tar command

Added compressing folder tmp/guesanumber/ to gzip archive quesanumber.tar.gz

Signed-off-by: Maxim Lipchanskyi <m4554747@gmail.com>

maximlip committed:
hometask01:part02: Add archive in folder release

Added creating folder release and coping archive to folder release

Signed-off-by: Maxim Lipchanskyi <m4554747@gmail.com>

maximlip committed:
hometask01.part02: Add refactoring with variables and loop

Added variables and loop into the script

Signed-off-by: Maxim Lipchanskyi <m4554747@gmail.com>

maximlip committed:
Lesson 2 Part 1: Add main() function implementation

Added implementation of base algorithm in main() function

Signed-off-by: Maxim Lipchanskyi <m4554747@gmail.com>

maximlip committed:
Lesson 2 Part 1: Add game() function

Added game() function as refactoring main() function

Signed-off-by: Maxim Lipchanskyi <m4554747@gmail.com>

a-vodka commented on 01-Git_Bash_Make/part2:
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+filelist="dz1.c part2"
+foldertmp="tmp/guesanumber"
```
might be /tmp/guesanumber

a-vodka commented on 01-Git_Bash_Make/part2:
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+filelist="dz1.c part2"
```
I think we need to copy all cpp files

a-vodka commented:
Please, look at my comments in the code

maximlip committed:
Lesson 2 Part 1: Remove return in main() function

Removed 2 returns as refactoring main() function

Signed-off-by: Maxim Lipchanskyi <m4554747@gmail.com>

maximlip committed:
hometask01:part02: Add refactoring in script

Changed tmp folder, added file mask *.c

Merge branch 'hometask01.part02' of https://github.com/maximlip/gl-kernel-basecamp-2019 into HEAD

Signed-off-by: Maxim Lipchanskyi <m4554747@gmail.com>
